tools/standardize_config_json.py

```diff
@@ -104,24 +103,57 @@ def print_diff(original: collections.OrderedDict, new: collections.OrderedDict):
     LOG.info("\n".join(diff))
 
 
-def process(filename: str):
-    contents = read_json_file(filename)
-    original = read_json_file(filename)
+def process(config_json_path: str, configuration: dict):
+    contents = read_json_file(config_json_path)
+    original = read_json_file(config_json_path)
 
     reordered = reorder_service(contents["properties"]["service"]["properties"])
     contents["properties"]["service"]["properties"] = reordered
 
+    sections = configuration.get("sections", {})
+    for section_name, head_and_tail in sections.items():
+        head = head_and_tail["head"]
+        tail = head_and_tail["tail"]
+        properties = contents["properties"][section_name]["properties"]
+        reordered = reorder(properties, head, tail, reorder_property)
+        contents["properties"][section_name]["properties"] = reordered
+
     print_diff(original, contents)
 
-    write_json_file(filename, contents)
+    write_json_file(config_json_path, contents)
 
 
 if __name__ == "__main__":
-    files = [
-        "frameworks/cassandra/universe/config.json",
-        "frameworks/hdfs/universe/config.json",
-        "frameworks/helloworld/universe/config.json",
-    ]
+    args = sys.argv[1:]
+
+    if args[0] in set(["-h", "-help", "--help"]):
+        print(usage)
+        exit(0)
+
+    if len(args) > 2:
+        print(usage)
+        exit(0)
+
+    config_json_path = args[0]
+
+    if not os.path.isfile(config_json_path):
+        LOG.info("'%s' is not a file, was expecting a config.json file", config_json_path)
+
+    configuration_path = None
+    configuration: dict = {}
+    LOG.info("config_json_path: %s", config_json_path)
+
+    if len(args) == 2:
+        configuration_path = args[1]
+        LOG.info("configuration_path: %s", configuration_path)
+
+    if configuration_path:
+        if not os.path.isfile(configuration_path):
```

Comment:
The two parameters are handled in similar ways. What about a "try_load" function? Also, should the following not log an error and terminate?

Reply:
The script should not terminate when not given a "SDK Tools" configuration file. In that case it standardizes just the `service` section.

Reply:
Also: deb7333e0f92c50103625f85493b0f5a5d4cc48f